drivers/include/vcnl40x0.h

```diff
@@ -0,0 +1,149 @@
+﻿/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_vcnl40x0 VCNL40X0
+ * @ingroup     drivers_sensors
+ * @brief       Device driver interface for the VCNL40X0 sensors family.
```

Comment:
this is not very descriptive, please add something like "... VCNL40x0 Proximity and Ambient Light Sensors ...".